eclass/vala.eclass

```diff
@@ -50,13 +50,16 @@ vala_api_versions() {
 	local minimal_supported_minor_version minor_version
 
 	# Dependency atoms are not generated for Vala versions older than 0.${minimal_supported_minor_version}.
-	minimal_supported_minor_version="22"
+	minimal_supported_minor_version="26"
 
 	for ((minor_version = ${VALA_MAX_API_VERSION#*.}; minor_version >= ${VALA_MIN_API_VERSION#*.}; minor_version = minor_version - 2)); do
 		if ((minor_version >= minimal_supported_minor_version)); then
 			echo "0.${minor_version}"
 		fi
 	done
+
+	# For some reason 0.22 is still in the tree but 0.24 is not
+	echo "0.22"
```

Comment:
I'm pretty sure this is wrong. The description says 'from VALA_MAX_API_VERSION down to VALA_MIN_API_VERSION'.